app/controllers/application_controller.rb

```diff
@@ -1374,6 +1374,11 @@ def find_checked_items(prefix = nil)
     end
   end
 
+  def find_checked_items_with_rbac(db, prefix = nil)
+    items = find_checked_items(prefix)
+    items.select { |item| find_by_id_filtered(db, item) }
```

Comment:
assert_rbac 👍 
but I believe that `user` parameter can be removed, RBAC will take logged user in other PR first. 

my imagination of new rbac_asser is something like this:

```